qa/workunits/cephtool/test.sh

```diff
@@ -1426,11 +1426,20 @@ function test_mon_osd()
   ceph osd blocklist ls | grep $bl
   ceph osd blocklist rm $bl
   ceph osd blocklist ls | expect_false grep $bl
-  expect_false "ceph osd blocklist $bl/-1"
-  expect_false "ceph osd blocklist $bl/foo"
+  expect_false "ceph osd blocklist add $bl/-1"
+  expect_false "ceph osd blocklist add $bl/foo"
 
-  # test with wrong address
-  expect_false "ceph osd blocklist 1234.56.78.90/100"
+  # test with invalid address
+  expect_false "ceph osd blocklist add 1234.56.78.90/100"
+
+  # test range blocklisting
+  bl=192.168.0.1:0/24
+  ceph osd blocklist range add $bl
+  ceph osd blocklist ls | grep $bl
+  ceph osd blocklist range rm $bl
+  ceph osd blocklist ls | expect_false grep $bl
+  bad_bl=192.168.0.1/33
+  expect_false ceph osd blocklist range add $bad_bl
```

Comment:
@gregsfortytwo I'm seeing this in the rados suite:

http://pulpito.front.sepia.ceph.com/yuriw-2022-06-01_02:28:14-rados-wip-yuri-testing-2022-05-31-1642-quincy-distro-default-smithi/6856753/
```
2022-06-01T03:34:44.256 INFO:tasks.workunit.client.0.smithi066.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1437: test_mon_osd:  ceph osd blocklist range add 192.168.0.1/24
2022-06-01T03:34:45.650 INFO:tasks.workunit.client.0.smithi066.stderr:blocklisting cidr:192.168.0.1:0/24 until 2022-06-01T04:34:44.774003+0000 (3600 sec)
2022-06-01T03:34:45.663 INFO:tasks.workunit.client.0.smithi066.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1438: test_mon_osd:  ceph osd blocklist ls
2022-06-01T03:34:45.663 INFO:tasks.workunit.client.0.smithi066.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1438: test_mon_osd:  grep 192.168.0.1/24
2022-06-01T03:34:46.112 INFO:tasks.workunit.client.0.smithi066.stderr:listed 7 entries
2022-06-01T03:34:46.122 INFO:tasks.workunit.client.0.smithi066.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1: test_mon_osd:  rm -fr /tmp/cephtool.kqr
2022-06-01T03:34:46.124 DEBUG:teuthology.orchestra.run:got remote process result: 1
2022-06-01T03:34:46.124 INFO:tasks.workunit:Stopping ['cephtool'] on client.0...
2022-06-01T03:34:46.125 DEBUG:teuthology.orchestra.run.smithi066:> sudo rm -rf -- /home/ubuntu/cephtest/workunits.list.client.0 /home/ubuntu/cephtest/clone.client.0
2022-06-01T03:34:46.380 ERROR:teuthology.run_tasks:Saw exception from tasks.
Traceback (most recent call last):
```
Has this been fixed in the new commits you added?

Reply:
This does contain the cephtool fix. I checked a subsequent rerun after the branch was rebased, and this problem does not appear anymore.